testStrategies/commonToAllTestStrategies.go

```diff
@@ -29,14 +29,11 @@ const (
 	SuiteBasedTesting   = "SuiteBased"
 )
 
-// GlobalsMaps maintains values across concurrent threads.
-type GlobalsMaps struct {
-	sync.RWMutex
-	m map[string]map[string]interface{}
-}
+// Global Mutex
+var mu sync.Mutex
 
```

Comment:
Putting mutex in a struct doesn't buy us anything since we still have to explicitly wrap every read and write in lock/unlock. It also tends to obfuscate the mutex. The global mutex (named "mu") follows the pattern established in golang docs.